core/shared/src/main/scala/me/shadaj/scalapy/interpreter/CPythonInterpreter.scala

```diff
@@ -309,6 +311,39 @@ object CPythonInterpreter {
     ), java.nio.charset.Charset.forName("UTF-8"))
   }
 
+  private object Traceback {
+    private var moduleOpt = Option.empty[py.Module]
```

Comment:
Within `CPythonInterpreter`, we should avoid using high-level `py.` APIs and instead use the lower-level primitives instead. This helps avoid tricky-to-debug circular dependencies. How we grab `weakRefModule` is a good reference for how do do things low-level.